### PR TITLE
Notifications update

### PR DESCRIPTION
**Pull request by benkeen**
This updates the global notifications to use the same store as the notification panel and drop Backbone.

### CODE REVIEW

**Comment by benkeen** [comment]
The license has been added to Couch here: https://github.com/apache/couchdb/pull/392

File: app/templates.js [hunk 1]
```diff
@@ -0,0 +1,623 @@
+this["JST"] = this["JST"] || {};
```
Comment (garrensmith) [inline app/templates.js]: This file shouldn't be here. you will have to do a `git rm`

**Comment by garrensmith** [comment]
This looks really cool, so awesome to see another piece of backbone disappear. Two things

* The animation looks a little jagged when a notification appears.
* The one test is failing - I think its worth just restarting the tests to see if its a flaky test.

Otherwise +1

File: app/templates.js (hunk 1 above)
Comment (benkeen) [inline app/templates.js]: Ah ok, thanks.

**Comment by benkeen** [comment]
@garrensmith - worrying about the animation looking jagged. Did it always look jagged for you, or just now and then? You're on Chrome/FF, I guess.

**Comment by garrensmith** [comment]
I'm on chrome. It always looked jagged.

**Comment by garrensmith** [comment]
+1

**Pull request merged**